packages/bitcore-node/src/modules/ethereum/p2p/p2p.ts

```diff
@@ -152,7 +152,7 @@ export class EthP2pWorker extends BaseP2PWorker<IEthBlock> {
         }
         disconnected = !connected;
       } catch (e) {}
-      await wait(2000);
+      await wait(10000);
```

Comment:
what's this change for?

Reply:
When not connected, this catch get's triggered. So it's every 2 seconds saying "Error syncing"
I altered it to 10, so it would be less log spam.

A better solution would probably be to stop trying to sync until it reconnects.